openwisp_utils/measurements/tasks.py

```diff
@@ -0,0 +1,55 @@
+import logging
+
+from celery import shared_task
+from openwisp_utils.admin_theme.system_info import (
+    get_enabled_openwisp_modules,
+    get_openwisp_version,
+    get_os_details,
+)
+
+from ..utils import retryable_request
+from .models import OpenwispVersion
+from .utils import _get_events, get_openwisp_module_events, get_os_detail_events
+
+CLEAN_INSIGHTS_URL = 'https://analytics.openwisp.io/cleaninsights.php'
+MAX_TRIES = 3
```

Comment:
what about setting this to 10? I don't see this as being used anywhere BTW. Are you sure it's not a leftover?